osu.Game/Overlays/Comments/VotePill.cs

```diff
@@ -145,16 +148,30 @@ protected override void OnLoadFinished()
 
         protected override bool OnHover(HoverEvent e)
         {
+            if (disabled)
+                return false;
```

Comment:
I'm not sure if I wouldn't prefer `true` to be returned here to prevent events from propagating further. It doesn't seem to interfere with drag in my testing.